programs/marginfi/src/errors.rs

```diff
@@ -23,8 +23,8 @@ pub enum MarginfiError {
     MissingBankAccount,
     #[msg("Invalid Bank account")] // 6009
     InvalidBankAccount,
-    #[msg("Bad account health")] // 6010
-    BadAccountHealth,
+    #[msg("RiskEngine rejected due to either bad health or stale oracles")] // 6010
+    RiskEngineInitRejected,
```

Comment:
Errors generally should refer to 1 condition if possible, here this error represents either bad health or stale oracles. Is it possible to make 2 new errors here and precisely identify the cause of the error?

Reply:
The behavior of ignoring stale oracles makes it impossible to determine wether its one or the other conclusively.

Reply:
In case there is ever any retroactive analysis of transactions, it may be worth considering leaving the old error as dead code and introducing this error with a new code. There are pros and cons to this approach -- off-chain and cpi clients may break if handling this error explicitly.

Reply:
Any client should actually deserialize this into the same error as before, as it still encodes it into the same enum representation